projects/ngrx.io/content/guide/signals/signal-store/testing.md

```diff
@@ -0,0 +1,687 @@
+<!-- TOC -->
```

Comment:
To make this page visible in the side nav, we should add link to the `navigation.json`: https://github.com/ngrx/platform/blob/main/projects/ngrx.io/content/navigation.json#L311